wormhole-connect/src/utils/routes/bridge.ts

```diff
@@ -29,9 +29,18 @@ import {
   TransferDestInfoBaseParams,
   TransferInfoBaseParams,
 } from './types';
-import { hexlify } from 'ethers/lib/utils.js';
 import { isCosmWasmChain } from '../cosmos';
 import { fetchVaa } from '../vaa';
+import { hexlify } from 'ethers/lib/utils.js';
+
+export interface BridgePreviewParams {
```

Comment:
Should this interface be here? I don't see it used anywhere else

Reply:
This shouldn't be here, removed